huggingface_sb3/push_to_hub.py

```diff
@@ -442,36 +433,26 @@ def push_to_hub(
     :param local_repo_path: local repository path
     """
 
-    temp = repo_id.split("/")
-    organization = temp[0]
-    repo_name = temp[1]
-
-    # Step 1: Clone or create the repo
-    # Create the repo (or clone its content if it's nonempty)
-    api = HfApi()
-    repo_url = api.create_repo(
-        name=repo_name,
+    repo_url =  HfApi().create_repo(
+        repo_id=repo_id,
         token=token,
-        organization=organization,
         private=False,
         exist_ok=True,
     )
 
-    # Git pull
-    repo_local_path = Path(local_repo_path) / repo_name
-    repo = Repository(repo_local_path, clone_from=repo_url, use_auth_token=True)
-    repo.git_pull(rebase=True)
-
     # Add the model
-    filename_path = os.path.abspath(filename)
-    _copy_file(Path(filename_path), repo_local_path)
-    _save_model_card(repo_local_path, "", {})
-
-    msg.info(f"Pushing repo {repo_name} to the Hugging Face Hub")
-    repo.push_to_hub(commit_message=commit_message)
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        filename_path = os.path.abspath(filename)
+        _copy_file(Path(filename_path), tmpdirname)
+        _save_model_card(tmpdirname, "", {})
+
+    msg.info(f"Pushing repo {repo_id} to the Hugging Face Hub")
+    repo_url = upload_folder(
+            repo_id=repo_id,
+            commit_message=commit_message,
+            token=token
+        )
```

Comment:
This call seems incorrect (missing the `folder_path` and `path_in_repo` params, and it's out of the TemporaryDirectory context )

Reply:
Yes, thanks! I haven't finished the push_to_hub, just package_to_hub function :+1: will update it, thanks for flagging!